templates/shared/forms/interactive/azure-1604.html

```diff
@@ -0,0 +1,191 @@
+    </header>
+    <div class="js-pagination js-pagination--1">
+      <p id="modal-description" class="u-no-max-width u-no-margin--bottom">If you are running Ubuntu 16.04 LTS on Microsoft Azure and have questions about upgrading or migrating your estate, please share some details with us and let's get started.</p>
+      <div class="js-formfield">
+        <h3 class="p-heading--five">Tell us more about your 16.04 instances</h3>
+        <div class="row u-no-padding">
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="about-your-1604-instances-less-than--50-vms">
+            <label for="about-your-1604-instances-less-than--50-vms">< 50 VMs</label>
+            <input type="checkbox" id="about-your-1604-instances-less-than-500-vms">
+            <label for="about-your-1604-instances-less-than-500-vms"><500 VMs</label>
+              <input type="checkbox" id="about-your-1604-instances--greater-than-500-vms">
+              <label for="about-your-1604-instances--greater-than-500-vms"> >500 VMs</label>
+              <input type="checkbox" id="about-your-1604-instances-containers">
+              <label for="about-your-1604-instances-containers">Containers</label>
+            </div>
+            <div class="col-4 u-sv3">
+              <input type="checkbox" id="about-your-1604-instances-azure-only-vms">
+              <label for="about-your-1604-instances-azure-only-vms">Azure only VMs</label>
+              <input type="checkbox" id="about-your-1604-instances-azure-only-aks">
+              <label for="about-your-1604-instances-azure-only-aks">Azure only AKS</label>
+              <input type="checkbox" id="about-your-1604-instances-multiple-public-clouds">
+              <label for="about-your-1604-instances-multiple-public-clouds">Multiple public clouds</label>
+              <input type="checkbox" id="about-your-1604-instances-private-or-edge-cloud">
+              <label for="about-your-1604-instances-private-or-edge-cloud">Private or edge cloud</label>
+            </div>
+            <div class="col-4 u-sv3">
+              <input type="checkbox" id="about-your-1604-instances-development-and-ci/cd">
+              <label for="about-your-1604-instances-development-and-ci/cd">Development and CI/CD</label>
+              <input type="checkbox" id="about-your-1604-instances-standard-production">
+              <label for="about-your-1604-instances-standard-production">Standard Production</label>
+              <input type="checkbox" id="about-your-1604-instances-mission-critical-production-regulated-workloads">
+              <label for="about-your-1604-instances-mission-critical-production-regulated-workloads">Mission-critical Production Regulated workloads</label>
+
+            </div>
+          </div>
+        </div>
+
+        <div class="js-formfield">
+          <h3 class="p-heading--five">Which of the following topics are important to you?</h3>
```

Comment:
I think this section would look more even if it was split into 3 cols of 6 checkboxes each. (It's fine as it is though)